package.json

```diff
@@ -54,8 +54,9 @@
     "patch-package": "6.4.7",
     "prettier": "2.4.1",
     "ts-jest": "27.0.7",
-    "typedoc": "0.21.9",
-    "typedoc-plugin-markdown": "3.9.0",
+    "typedoc": "0.22.9",
+    "typedoc-plugin-markdown": "3.11.6",
+    "typedoc-plugin-rename-defaults": "0.4.0",
```

Comment:
to fix this
![image](https://user-images.githubusercontent.com/7361780/142673150-8a6bddb0-48c3-4c34-ab11-2031af76f0c6.png)